.github/workflows/sam-pull-request.yml

```diff
@@ -0,0 +1,43 @@
+name: Pre-merge checks
+
+on:
+  pull_request:
```

Comment:
Does this also need to happen for merge queues?

Reply:
Will do